servicebox-app/src/main/java/io/halkyon/model/Claim.java

```diff
@@ -43,9 +49,17 @@ public Claim(String name, String serviceRequested, String description, String st
     public String status;
     @FormParam
     public String owner;
-
     @FormParam
     public Date created;
+    @OneToOne(cascade = CascadeType.ALL)
+    public ClaimingStatus claimingStatus;
```

Comment:
I would prefer to rename this one, having claimingStatus and status in the same entity is not helpful

Reply:
I agree... I will look for another name.

Reply:
StatusBinding? BindingStatus?

Reply:
The new table is now gone. I moved the `attempts` field from the deleted ClaimingStatus entity to the Claim entity.